scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json

```diff
@@ -237,7 +237,7 @@
                           "mode": "Incremental",
                           "id": "[parameters('aksResourceId')]",
                           "addonProfiles": {
```

Comment:
No changes required. This has to be omsagent for AKS.

Reply:
removed